ecosystem/Editor-and-tool-support.md

```diff
@@ -74,7 +74,7 @@ There are many tools available to help you develop libraries and applications in
 
 There are several build tools and package managers available for PureScript.
 
-- [spago](https://github.com/purescript/spago) is the standard package manager and build tool for Purescript, powered by [Dhall](https://github.com/dhall-lang/dhall-lang) and [package-sets](https://github.com/purescript/package-sets).
+- [spago](https://github.com/purescript/spago) is the standard package manager and build tool for Purescript.
```

Comment:
Perhaps this should link to `spago-legacy` because the Spago rewrite is still alpha?

Reply:
I considered this, but I went by the fact `spago-legacy` is officially deprecated and the repo is occupied by the Spago rewrite, which would by extension mean that most of the ecosystem links to the Spago rewrite. Since this page is aimed at first-comers I decided it's worth it to just link to new Spago somewhat ahead of time to avoid extra confusion. If your suggestion is a definitive one I could revert it.

Reply:
I think we should highlight here that the `spago` repo is still in alpha and most learning resources assume `spago-legacy`, even though it's been deprecated. Most people looking at these files will be new learners. So, we don't want to complicate their learning journey by installing the tool most resources don't use.

Reply:
Good points, thanks. I updated the document to link to `spago-legacy` instead.

Reply:
Oh, that's not what I meant. I think both should be linked to in the same section but with clarification on the situation.

Reply:
Oh sorry then, I misunderstood it. I'll link to both and add a note with clarification now.